desktop/apps/auction2/components/range_slider/rc_slider.styl

```diff
@@ -0,0 +1,290 @@
+.rc-slider {
```

Comment:
Looks like [their example](https://github.com/react-component/slider#usage) mentions importing in JS (presuming some Babel transform I suppose). You could do essentially the same thing with stylus usin g`@require`.